enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -0,0 +1,100 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static admin_urls %}
+
+{% block extrastyle %}
+<!-- TODO: find a way to extract it into a file and make edx-platform find that file -->
```

Comment:
@saleem-latif I've tried doing something like this, but was unsuccessful. Not exactly what you're suggesting, so maybe it'll help. I'll give it a try (but tomorrow, I'm sure I can't fit it today).